include/sourmash.h

```diff
@@ -227,6 +227,8 @@ uint32_t kmerminhash_num(const SourmashKmerMinHash *ptr);
 
 void kmerminhash_remove_hash(SourmashKmerMinHash *ptr, uint64_t h);
 
+void kmerminhash_remove_from(SourmashKmerMinHash *ptr, const SourmashKmerMinHash *other);
+
```

Comment:
Functions in `include/sourmash.h` need to be in lexicographical order =]

```suggestion
void kmerminhash_remove_from(SourmashKmerMinHash *ptr, const SourmashKmerMinHash *other);

void kmerminhash_remove_hash(SourmashKmerMinHash *ptr, uint64_t h);

```